project/src/main/java/com/google/sps/servlets/EventCreationServlet.java

```diff
@@ -62,4 +62,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     // Event in database
     response.getWriter().println(CommonUtils.convertToJson(SpannerTasks.getEventById(event.getId()).get().toBuilder().build()));
   }
+  private static List<String> splitAsList(String values) {
```

Comment:
Feel free to move this to CommonUtils because I think it's the second declaration of this function

Reply:
will be done when others get pushed to master